FirebaseAI/Sources/Types/Public/Live/LiveSession.swift

```diff
@@ -0,0 +1,140 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// A live WebSocket session, capable of streaming content to and from the model.
+///
+/// Messages are streamed through ``responses``, and can be sent through either the dedicated
+/// realtime API function (such as ``sendAudioRealtime(audio:)`` or ``sendTextRealtime(text:)``), or
+/// through the incremental API (such as ``sendContent(_:turnComplete:)``).
+///
+/// To create an instance of this class, see ``LiveGenerativeModel``.
+@available(iOS 15.0, macOS 12.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
+@available(watchOS, unavailable)
+public final class LiveSession: Sendable {
```

Comment:
@daymxn , is it expected that all of the APIs in this file don't throw? In the API proposal, they were all marked as throwing.

Reply:
Yes. The API proposal needs updating. The errors surface from the `responses` stream and setup task via `connect`.